python/docs/source/reference/pyspark.sql.rst

```diff
@@ -72,6 +72,11 @@ See also :class:`SparkSession`.
     SparkSession.udf
     SparkSession.version
 
+.. toctree::
+    :hidden:
+
+    pyspark.sql.conf.RuntimeConfig
```

Comment:
Shall we just add another section "Configuration"? e.g.)

```
Configuration
-------------

.. currentmodule:: pyspark.sql.conf

.. autosummary::
    :toctree: api/

    RuntimeConfig
```

Reply:
Yes, Hyukjin, that will work. I was unsure if that should be elevated to its own section, but it is definitely a simpler / more elegant solution. Thank you for the suggestion.  I will update the PR today.

Reply:
I pushed up changes with your suggestion, removed the RuntimeConfig rst file to allow autosummary to generate it, and updated the PR description.  Thank you for the review, Hyukjin.